protx-client/src/components/ProTx/components/charts/MainPlot.module.scss

```diff
@@ -0,0 +1,20 @@
+.data-error-message {
+  margin: 1rem;
+  text-align: center;
+  font-weight: 700;
+  color: var(--color-blood-orange);
+}
+.loading-spinner {
+  margin: 5%;
+  margin-top: 40%;
+  text-align: center;
+  font-weight: 700;
+}
+.main-plot * {
```

Comment:
I have this now in two spots CSS and in scss. so needs to be fixed before merging.

Reply:
fixed by @taoteg PR to this branch